tensorflow/contrib/makefile/create_ios_frameworks.sh

```diff
@@ -45,6 +45,13 @@ cp $SCRIPT_DIR/gen/lib/libtensorflow-core.a \
    $FW_DIR_TFCORE/tensorflow_experimental
 cp $SCRIPT_DIR/gen/protobuf_ios/lib/libprotobuf.a \
    $FW_DIR_TFCORE/libprotobuf_experimental.a
+IOS_ARCH=$(ls $SCRIPT_DIR/gen/bin|grep ios|awk '{print tolower($0)}')
+NSYNC_PATH=./downloads/nsync/builds/${IOS_ARCH:4}.ios.c++11/libnsync.a
```

Comment:
I think this works only when building TensorFlow for one architecture? (`compile_ios_tensorflow.sh -a arm64`)
However it doesn't work when building for multiple architectures (`compile_ios_tensorflow.sh`)
In the latter case, the current script will archive the library at `downloads/nsync/builds/lipo.ios.c++11/nsync.a`

Could you test the multiple architectures flow?

Reply:
I am not sure, it is a bit complicated and I couldn't set up a full testing environment for all these architectures recently and it also seems to be a good time to TF lite?? Sorry about, apologies, maybe you can close this PR.

Reply:
Thanks for reporting this issue, @RockfordWei 
No need to apology, and I appreciate you for flagging the issue. 

I think you probably only need to change the line from 
    NSYNC_PATH=./downloads/nsync/builds/${IOS_ARCH:4}.ios.c++11/libnsync.a
to
    NSYNC_PATH=.//downloads/nsync/builds/lipo.ios.c++11/nsync.a
(See the bottom of the `compile_nsync.sh`. It uses `lipo` to archive nsync from all architectures to a single file)

Could you help to test it? Just run the scripts without `-a` parameters, and it will build for all architectures

    $ ./tensorflow/contrib/makefile/build_all_ios.sh
    $ ./tensorflow/contrib/makefile/create_ios_frameworks.sh

Try to use the framework on iOS device (arm) and/or emulator (normally it should be x64). You don't need to test all the architectures. 

Thanks a lot again!

Reply:
Hi Ling @miaout17, I just updated a bit, however, as I said, it is a bit complicated, since the simple `build_all_ios.sh` never works for me, I'd been using a long command as mentioned in [my own TF ios demo page](https://github.com/RockfordWei/TensorFlow-iOS-Demo-App) instead, for example (simulator):
``` bash
./build_all_ios.sh -a x86_64 -g $(PROJECT_DIR)/xxx.pb
```
if without these parameters, the lib cannot build at all and is throwing some local thread symbol missing error.


However, I can confirm that after this update, there is a `libnsync.a` in the expected framework folder, but still have no chance to all the ios frameworks, just one by one is ok.

Reply:
Thanks. This looks good to me. 

`build_all_ios.sh` is working for me. May I ask what's the error message?